Makefile

```diff
@@ -1,19 +1,22 @@
 ARCHS=linux linux64 mac64 win32
 
-ROOT_PATH=/pub/labs/fxos-simulator/adb-helper/
-ROOT_UPDATE_URL=https://ftp.mozilla.org$(ROOT_PATH)
+ADDON_NAME=adb-extension
+VERSION=0.0.1
+XPI_NAME=$(ADDON_NAME)-$(VERSION)
 
-VERSION=1.0
+ROOT_PATH=pub/labs/fxos-simulator/$(ADDON_NAME)
```

Comment:
Oh, adb-extension looks nice.  so the ROOT_PATH should be;

 pub/devtools/$(EXTENSION_NAME)

?

Reply:
I don't have access to pub/* , but I actually have access to pub/labs/*, so I should be able to make that pub/labs/devtools at least.